### DISCUSSION TITLE
Creating a custom dataset for text detection

### DISCUSSION THREAD

**Discussion by TomekPro** [mindee/doctr#1515] [General]
Hi, does somebody has any hints on creating own dataset to fine-tune detection model? The problem I have with current models is that for example in polish "and" is "i" and current models very often don't recognize "i" as a single word, glue it to the previous or consecutive word.
@felixdittrich92 in some thread I saw that at Mindee you have your own datasets, were they created in some automatic way? For example parsing pdfs?

Thanks
Tomek

**Comment by felixdittrich92** [comment]
Hi @TomekPro :wave: 

We talk about recognition instead of detection right ?

Yeah i have created a first version of a full synthetic dataset (multilingual created with a modified https://github.com/clovaai/synthtiger version) to fine tune the pretrained models (https://huggingface.co/Felix92/doctr-torch-parseq-multilingual-v1).

I can share the dataset if you want.

I have on a long term track to create a real multilingual detection/recognition dataset (pre labeled with Azure) but currently a lack of time to collect some free data with a wider distribution (magazines, papers, id cards, etc.)

Best regards,
Felix

**Reply by TomekPro** [reply]
Hi, unfortunately I mean detection model. My problem is the following: in such case as below "i" is not treated as separate text and as a result later on reco model will return "SKOKOWEGOI" AND "ISTOPY" in this case
![image](https://github.com/mindee/doctr/assets/49069202/51381c29-b74d-44c7-ace0-99de234fcae0)

I believe, that fine-tunning a detection model on polish language could help.

**Reply by whoami02** [reply]
@TomekPro Hello, I recently created a custom dataset for symbol detection. In my case I simply wanted the model to create bounding boxes along the text. I used opencv to manually create bounding boxes on the image and extract coordinates in a JSON file.

**Reply by TomekPro** [reply]
@whoami02  "I used opencv to manually create bounding boxes on the image and extract coordinates in a JSON file." - totally manually? this seems to be a lot of work to create big enough dataset :/
